src/components/label-pill/LabelPill.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import Tooltip, { TooltipPosition } from '../tooltip';
+
+import './LabelPill.scss';
+
+export enum StatusLevelType {
+    DEFAULT = 'default',
+    INFO = 'info',
+    HIGHLIGHT = 'highlight',
+    SUCCESS = 'success',
+    WARNING = 'warning',
+    ALERT = 'alert',
+    ERROR = 'error',
+}
+
+export interface LabelPillProps {
+    children: React.ReactNode;
+    tooltip?: string;
+    type?: StatusLevelType;
+}
+
+const LabelPill = ({ children, type = StatusLevelType.DEFAULT, tooltip = '', ...rest }: LabelPillProps) => {
+    return (
+        <Tooltip position={TooltipPosition.TOP_CENTER} text={tooltip} isDisabled={!tooltip}>
+            <span
+                className={classNames(classNames('bdl-LabelPill', type !== 'default' ? `bdl-LabelPill--${type}` : ''))}
```

Comment:
2 nested classNames? Should extract it out as a variable for readability and debuggability. Also default string should be changed to enum value.

Reply:
It can be this:

```js
  classNames('bdl-LabelPill', { [`bdl-LabelPill--${type}`]: type !== 'default' })
```

Reply:
Not sure if `type` is a reserved word in Typescript, I guess not

Reply:
For reference, it's useful to convert an enum to it's individual values, for use in storybook knobs. However, this appears to be some unspecified functionality in typescript. You can search for it and see all the confusion around it. Kinda nasty that there's no clear construct around this use case.

Just in case you're wondering why I have that manual array in the .stories for this...